app/src/main/java/fr/free/nrw/commons/media/Labels.java

```diff
@@ -0,0 +1,59 @@
+package fr.free.nrw.commons.media;
+import com.google.gson.annotations.SerializedName;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Labels {
+
+    @SerializedName("en")
+    private CaptionObject captionObject;
```

Comment:
@nicolas-raoul , this was added keeping in mind that I am sending my prefered language to the API in which I want my captions. Thus I believe that it will not return a list of captions.